_tutorials/2016-08-22-iosxr-puppet-and-vagrant.md

```diff
@@ -0,0 +1,460 @@
+---
+published: false
+date: '2016-08-22 18:29 -0400'
+title: 'IOSXR: Puppet And Vagrant'
```

Comment:
Let me suggest "Using Puppet with IOS-XR 6.1.1"  as the title. Will be more catchy on twitter.